deploy/kubernetes/releases/csi-digitalocean-dev.yaml

```diff
@@ -168,6 +168,19 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: csi-resizer
+          image: quay.io/k8scsi/csi-resizer:v0.3.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: "IfNotPresent"
```

Comment:
nit: FWIW, other image pull policies defined in our manifests use "Always". Not really sure what's better, but might be good to be consistent (unless we have a good reason to diverge).

Reply:
I was looking at the imagePullyPolicy for the existing containers. The images for them have versions as tags. So, I don't understand why the policy is set to `Always`.  When there is a change, we have to bump up the tag anyway. So, why download the image every time?

Reply:
That's a fair point, I think it makes sense to stick to `IfNotPresent` then. 👍 

(Technically, I believe `Always` doesn't pull the entire image but checks which layers are missing / diverging and only pull if that's the case. But still no good reason for us to use `Always` .)

Reply:
I'll raise a new PR to change other containers' imagePullPolicy to IfNotPresent. Is that okay?

Reply:
Sounds good to me! 👍